MasterForm.Designer.cs

```diff
@@ -183,325 +183,304 @@ partial class MasterForm {
+            this.MyModsTab.Controls.Add(this.homeEnabledLabel);
+            this.MyModsTab.Controls.Add(this.delModpack);
+            this.MyModsTab.Controls.Add(this.patchButton);
+            this.MyModsTab.Controls.Add(this.homeNameLabel);
+            this.MyModsTab.Controls.Add(this.homeSelectLabel);
+            this.MyModsTab.Controls.Add(this.modListPanel);
+            this.MyModsTab.Location = new System.Drawing.Point(0, 36);
+            this.MyModsTab.Name = "MyModsTab";
+            this.MyModsTab.Padding = new System.Windows.Forms.Padding(3);
+            this.MyModsTab.Size = new System.Drawing.Size(579, 366);
+            this.MyModsTab.TabIndex = 0;
+            this.MyModsTab.Text = "My Mods";
             // 
             // manualOverride
             // 
             this.manualOverride.AutoSize = true;
             this.manualOverride.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.25F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
             this.manualOverride.ForeColor = System.Drawing.Color.DarkRed;
-            this.manualOverride.Location = new System.Drawing.Point(289, 13);
+            this.manualOverride.Location = new System.Drawing.Point(294, 23);
             this.manualOverride.Name = "manualOverride";
             this.manualOverride.Size = new System.Drawing.Size(132, 17);
-            this.manualOverride.TabIndex = 8;
+            this.manualOverride.TabIndex = 16;
             this.manualOverride.Text = "Allow Manual Override";
             this.manualOverride.UseVisualStyleBackColor = true;
             // 
             // selectEnabled_chb
             // 
             this.selectEnabled_chb.AutoSize = true;
-            this.selectEnabled_chb.Checked = true;
-            this.selectEnabled_chb.CheckState = System.Windows.Forms.CheckState.Checked;
-            this.selectEnabled_chb.Location = new System.Drawing.Point(434, 247);
+            this.selectEnabled_chb.Location = new System.Drawing.Point(439, 234);
             this.selectEnabled_chb.Name = "selectEnabled_chb";
-            this.selectEnabled_chb.Size = new System.Drawing.Size(98, 17);
-            this.selectEnabled_chb.TabIndex = 7;
+            this.selectEnabled_chb.Size = new System.Drawing.Size(112, 17);
+            this.selectEnabled_chb.TabIndex = 15;
             this.selectEnabled_chb.Text = "Select Enabled";
             this.selectEnabled_chb.UseVisualStyleBackColor = true;
             // 
             // homeEnabledLabel
             // 
             this.homeEnabledLabel.AutoSize = true;
-            this.homeEnabledLabel.Font = new System.Drawing.Font("Reem Kufi", 8.999999F);
-            this.homeEnabledLabel.Location = new System.Drawing.Point(10, 7);
+            this.homeEnabledLabel.Font = new System.Drawing.Font("Microsoft Sans Serif", 8.999999F);
+            this.homeEnabledLabel.Location = new System.Drawing.Point(15, 17);

```

Comment:
The labels should still be centered over the icons and in line with the modpack name. This is probably a product of using the tab control in that everything is shifted, so there's probably not an issue her. I seem to recall that none of the alignment looked off in your screenshots, I'd just like you to double check this for all of the tab pages

Reply:
Will revert

Reply:
This is correct. After reviewing, everything on the tab controls is shifted by a mere 3 pixels. I reverted the top bar, but once I saw that everything was shifted, I opted not to do it by hand. Lots of tedium

It's not noticeable, and I don't believe it will affect your location code as it behaves relative to other items. And if those items are also shifted by 3 pixels, then there's no problem.

I can send you screenshots of how it looks, or you can pull this branch and look for yourself.